librz/arch/p/analysis/analysis_pic.c

```diff
@@ -100,7 +100,7 @@ RzAnalysisPlugin rz_analysis_plugin_pic = {
 	.desc = "PIC analysis plugin",
 	.license = "LGPL3",
 	.arch = "pic",
-	.bits = 8,
+	.bits = 32,
```

Comment:
Shouldn't it be multiple values? `.bits = 8 | 16 | 32`? See how it's done in `analysis_mips.c` Same for the asm plugin.